core/__tests__/bin/apply.ts

```diff
@@ -43,8 +43,8 @@ describe("bin/config-apply", () => {
     resetSettings: true,
   });
 
-  let messages = [];
-  let spies = [];
+  let messages: string[] = [];
+  const spies: any[] = [];
```

Comment:
Sometimes an `any` is OK... but you need to call it out.  In tests where things are getting mocked/spied this seems OK

Reply:
For this case I believe you could do `const spies: jest.SpyInstance[]`

Reply:
Nice! updated in aafc187a9